std-bits/table/src/main/java/org/enso/table/data/column/builder/LongBuilder.java

```diff
@@ -108,16 +108,11 @@ public void appendBulkStorage(Storage<?> storage) {
    * @param data the integer to append
    */
   public void appendLong(long data) {
-    int wasSize = currentSize;
-    int wasLength = this.data.length;
-
     if (currentSize >= this.data.length) {
       grow();
     }
 
-    if (currentSize >= this.data.length) {
-      throw new IllegalStateException("currentSize=" + currentSize + "; wasSize=" + wasSize + "; wasLength=" + wasLength + "; data.length=" + this.data.length);
-    }
+    assert currentSize < this.data.length;
```

Comment:
appreciate this shouldn't happen but as asserts don't run in Enso we won't check this any more.

Reply:
I know. This was a debug check that I added in a past PR and forgot to remove back then.

The assert is kept there for 'documentation purposes'. Current code guarantees it should not be violated.